pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -70,16 +70,23 @@
     @Parameter(names = { "-c", "--config" }, description = "Configuration file path", required = true)
     private String configFile;
 
+    @Deprecated
     @Parameter(names = { "-zk", "--zookeeper-servers" }, description = "Local zookeeper connection string")
```

Comment:
Could you explain which is the successor of this parameter?

Reply:
done @nicoloboschi